docs/docs/20-usage/20-pipeline-syntax.md

```diff
@@ -412,8 +412,7 @@ when:
 
 :::info
 Path conditions are applied only to **push** and **pull_request** events.
-It is currently **only available** for GitHub, GitLab.
-Gitea only supports **push** at the moment ([go-gitea/gitea#18228](https://github.com/go-gitea/gitea/pull/18228)).
+It is currently **only available** for GitHub, GitLab and Gitea.
```

Comment:
could you clarify that 'sed s/Gitea/Gitea[1]/g' ...

[1] all gitea version >= 1.18.0 ...